client/lib/plugins/wporg-data/test/actions.js

```diff
@@ -32,6 +33,10 @@ describe( 'WPorg Data Actions', () => {
 		assert.isFunction( WPorgActions.fetchPluginsList );
 	} );
 
+	it( 'Actions should have method fetchCuratedList', () => {
+		assert.isFunction( WPorgActions.fetchCuratedList );
```

Comment:
Just a heads up that most test files, do use `expect` over `assert`. We don't have any hard rules for this, so I think it's fine to keep the file consistent here. In any case we're usually more interested that our cases remain descriptive of behavior over testing an arbitrary implementation.